### DISCUSSION TITLE
Land v1.2 [RELEASE]

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#160] [Map Releases]
## This is a manual migration of forum user oldschool125's map release `land`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t1982-land-v1-2-release).

Hi guys today i'm gonna show the hard work that we was working on me and Naievil, Hope u guys enjoy !
Here is a trailer of the map 😉, as you can see in this video we can surround with zombies :
https://www.youtube.com/watch?v=r7U2Co4ZIz4
Screenshot:

![Pq2V2Gs](https://user-images.githubusercontent.com/26030250/216119683-55c25995-02f8-452f-825e-200b5c1f97f5.jpeg)

I don't want to spoil anymore 😄 here is for u

Note:Don't ask me how to reach the flooper,it is just a model that i putted in.And this map is working on psp ,u will just need the 1.0.2 unofficial patch

Features:

Added more waypoints
Added a lot of models
Changed layout

More maps coming soon [anass-nzp125],[Naievil]

Here is a preview video of my map,this video was made before fixing the way 😉 :

https://www.youtube.com/watch?v=ZQCQ5f0RrLA

# [DOWNLOAD](https://archive.org/download/nzp_land/nzp_land.zip)

**Comment by SgtCookieBobble** [comment]
Really enjoy this map - feels like a map that would be on the DS version of Call of Duty. My only issue is the box selection. I thought it was my files, but was able to get good guns on other maps. Felt like the MG42 was the only helpful weapon.
